framework/meta/src/cmd/standalone/template/contract_creator.rs

```diff
@@ -31,10 +32,10 @@ fn target_from_args(args: &TemplateArgs) -> ContractCreatorTarget {
 
 pub(crate) fn get_repo_version(args_tag: &Option<String>) -> RepoVersion {
     if let Some(tag) = args_tag {
-        assert!(validate_template_tag(tag), "invalid template tag");
+                assert!(validate_template_tag(tag), "invalid template tag");
         RepoVersion::Tag(tag.clone())
     } else {
-        RepoVersion::Tag(LAST_TEMPLATE_VERSION.to_string())
+        RepoVersion::Tag(LAST_TEMPLATE_VERSION.version.to_string())
```

Comment:
If you implement `Display` for `FrameworkVersion`, you can call to_string() on it directly.